src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  @extend %grid-center;
+  @include template-columns(1);
+  gap: 48px;
+
+  @media (min-width: 488px) {
+    @include template-columns(2);
+  }
+
+  @media (min-width: 768px) {
+    @include template-columns(3);
+  }
+
+  @media (min-width: 1024px) {
+    @include template-columns(4);
+  }
+
+  &__card {
+    @extend %grid-center;
+    align-items: center;
+    width: $card-width;
```

Comment:
wouldn't reject, because I like your solution overall, but you don't need width here, width of your card should be controlled by grid
```suggestion
```